modules/microshift-cni-customization-matrix.adoc

```diff
@@ -0,0 +1,45 @@
+
+:_mod-docs-content-type: REFERENCE
+[id="microshift-nw-ovn-kubernetes-matrix_{context}"]
+= {microshift-short} CNI feature matrix
+
+The following table summarizes the status of OVN-K networking features that are either present as defaults, supported for configuration, or excluded from the {microshift-short} service:
+
+.Default {microshift-short} OVN-K network plugin features and customization status
+[cols="50%,20%,30%",options="header"]
+|===
+|Network feature|Availability|Customization status
+
+|Kubernetes network policy|Yes|Supported
+
+|Load balancing|Yes|Supported
+
+|Multicast|Yes|mDNS supported
+
+|Network proxies|Yes ^[2]^|CRI-O supported
+
+|Network performance|Yes|MTU configuration supported
+
+|Egress IPs|Not available|N/A
+
+|Egress firewall|Not available|N/A
+
+|Egress router|Not available|N/A
+
+|Firewall|No ^[1]^|Supported
+
+|Hardware offloading|Not available|N/A
+
+|Hybrid networking|Not available|N/A
+
+|IPsec encryption for intra-cluster communication|Not available|N/A
+
+|IPv6||Not available|N/A
+
+|Kubernetes network policy logs|Not available|N/A
+|===
+
+1. Setting up a firewall is supported by the OVN-K CNI.
```

Comment:
I think these and the numbers in the table should be switched just to be sequential cuz in the table rn, 2 comes first